src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -268,8 +268,8 @@ export class CalciteRadioButton {
   connectedCallback(): void {
     this.guid = this.el.id || `calcite-radio-button-${guid()}`;
     this.initialChecked = this.checked;
-    this.renderInput();
     this.renderLabel();
+    this.renderInput();
```

Comment:
These had to be reordered since `renderLabel` now removes all children first before adding the final concatenated result.